doc/CONFIGURATION.md

```diff
@@ -1,6 +1,20 @@
 # Configuration File Documentation
 
-TemporalX configuration is done through a yaml file, while the command line `tex-cli` has a helper command to generate a fully populated config file.
+TemporalX configuration is done through a yaml file, while the command line tool `tex-cli` has a commad `tex-cli config` that can be used to generate default configuration files.
```

Comment:
... done through a yaml file located at [location]

Reply:
There is no specific location requirement for the config file, it can be located anywhere.

Reply:
But that's the first question I would ask. If it can be anywhere, then how the location is set should be stated.

Reply:
good point